Makefile

```diff
@@ -25,6 +25,14 @@ check:
 	@echo "*** Running pylint ***"
 	PYTHONPATH=$(PYTHONPATH):./src/ ./tests/pylint/runpylint.py
 
+test:
+	@echo "*** Running tests ***"
+	PYTHONPATH=$(PYTHONPATH):./src/ $(PYTHON) -m nose -v --with-coverage --cover-erase --cover-branches \
+                        --cover-package=pylorax --cover-package=bin --cover-package=sbin --cover-inclusive
+	coverage3 report -m
+	[ -f "/usr/bin/coveralls" ] && [ -n "$(COVERALLS_REPO_TOKEN)" ] && coveralls || echo
```

Comment:
Same question here as on lorax-composer branch, although I'm not as concerned about using 3rd party stuff for Fedora builds. Where does coveralls come from

Reply:
That's python3-coveralls-1.1-4.fc26.noarch